build.sbt

```diff
@@ -138,7 +138,8 @@ val testSettings = Seq(
   testOptions in Test += Tests.Setup(_ => {
 
     println(s"Launching docker container with ES")
-    s"docker-compose up -d elasticsearch".!
+    s"docker-compose --file docker-compose-test.yml --project-name grid-test down".!
```

Comment:
Deleting the docker stack appears to be key to getting a green build. An alternative could be to assign a random `project-name` each time, but that may result in random stacks remaining if not cleaned up properly.